client/src/pages/private/PSIView.js

```diff
@@ -108,14 +108,15 @@ export default () => {
         const data = await response.json();
         const mappedData = data.map((row) => {
           const rowCohorts = cohorts.filter((cohort) => cohort.psi_id === row.id);
```

Comment:
Unrelated to this PR, but this looks like we're getting all cohorts from the API and then reducing to the current PSI, why not pass the PSI ID to the API and only retrieve the records we want? It looks like there's logic written on the API to retrieve a specific PSI's cohorts.

Reply:
In the case of the PSIView, we actually do want all of the cohorts since we're using all of the PSIs. It could probably be done more efficiently but nothing really jumped out at me.